pkg/git/fetch_test.go

```diff
@@ -32,14 +31,21 @@ func testGitCheckout(t *testing.T, when spec.G, it spec.S) {
 
 		it.Before(func() {
 			var err error
-			testDir, err = ioutil.TempDir("", "test-git")
+			testDir, err = os.MkdirTemp("", "test-git")
 			require.NoError(t, err)
 
-			metadataDir, err = ioutil.TempDir("", "test-git")
+			metadataDir, err = os.MkdirTemp("", "test-git")
 			require.NoError(t, err)
+			if _, exists := os.LookupEnv("HTTPS_PROXY"); exists {
+				require.NoError(t, os.Unsetenv("HTTPS_PROXY"))
+			}
 		})
 
 		it.After(func() {
+			if _, exists := os.LookupEnv("HTTPS_PROXY"); exists {
+				require.NoError(t, os.Unsetenv("HTTPS_PROXY"))
+			}
+			require.NoError(t, os.Setenv("HTTPS_PROXY", "http://invalid-proxy"))
```

Comment:
What is this line doing?

Reply:
48 It shouldn't be there. Removed. And I simplified the Unsetenv, no lookupenv necessary.